neuro_config_client/entities.py

```diff
@@ -61,6 +61,49 @@ class NodePool:
 
     zones: Sequence[str] = field(default_factory=tuple)
 
+    @property
+    def id(self) -> str | None:
+        if not self.machine_type:
+            return None
+        return get_node_pool_id(self.machine_type, self.cpu)
+
+
+@dataclass(frozen=True)
+class NodePoolTemplate:
+    machine_type: str
+    cpu: float
+    available_cpu: float
+    memory_mb: int
+    available_memory_mb: int
+    gpu: int | None = None
+    gpu_model: str | None = None
+
+    @property
+    def id(self) -> str:
+        return get_node_pool_id(self.machine_type, self.cpu)
+
+    def to_nodepool(self, name: str | None = None) -> NodePool:
+        return NodePool(
+            name=name or self.id,
+            machine_type=self.machine_type,
+            cpu=self.cpu,
+            available_cpu=self.available_cpu,
+            memory_mb=self.memory_mb,
+            available_memory_mb=self.available_memory_mb,
+            gpu=self.gpu,
+            gpu_model=self.gpu_model,
+        )
+
+
+def get_node_pool_id(machine_type: str, cpu: float) -> str:
```

Comment:
please remove it, client should receive id from server

Reply:
done in [44c6d27](https://github.com/neuro-inc/neuro-config-client/pull/83/commits/44c6d279d91f77f339668d2979e94a735354c35d)